dgraphql/resolvers/resolver.go

```diff
@@ -820,6 +820,24 @@ func (t *TransactionTrace) ExceptJSON() (*commonTypes.JSON, error) {
 	return nil, nil
 }
 
+type BlockRootMerkle struct {
+	m *pbcodec.BlockRootMerkle
+}
+
+func newBlockRootMerkle(merkleRoot *pbcodec.BlockRootMerkle) *BlockRootMerkle {
+	return &BlockRootMerkle{
+		m:  merkleRoot,
+	}
+}
+
+func (b BlockRootMerkle) NodeCount() commonTypes.Uint32                    { return commonTypes.Uint32(b.m.NodeCount) }
+func (b BlockRootMerkle) ActiveNodes() (out []string){
+	for _, n := range b.m.ActiveNodes{
+		out = append(out, hex.EncodeToString(n))
+	}
+	return
```

Comment:
Let's pre-allocate array with the right size upfront when the size is known and especially fixed
```suggestion
      out = make([]string, len(b.m.ActiveNodes))
	for i, n := range b.m.ActiveNodes{
		out[i] = hex.EncodeToString(n)
	}
	return
```